.github/workflows/coverage-diff.yml

```diff
@@ -0,0 +1,34 @@
+name: 'coverage-diff'
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  coverage-diff:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.2
+          coverage: none
+          tools: composer:v2
+          extensions: xdebug
+      - name: Install Dependencies
+        run: composer install -q --profile --ignore-platform-reqs --no-interaction --no-ansi --no-scripts --no-suggest --prefer-dist
+      - uses: KengoTODA/actions-setup-docker-compose@4677f0d86d41e623c9c6e11e1d910976da297bc0
+        with:
+          version: '2.14.2'
+      - name: Set Permissions for Coverage Directory
+        run: |
+          mkdir -p cov/json cov/xml cov/html
+          chmod -R 777 cov
+      - name: "Create Empty env File for Docker"
+        run: touch .env
+      - name: PHPUnit
+        run: docker-compose run phpunit --coverage-xml cov/xml --coverage-html cov/html --coverage-clover=cov/xml/clover.xml
+      - name: Get Cover
```

Comment:
I don't see the result of this action

Reply:
This is the action: https://github.com/fingerprintjs/fingerprint-pro-server-api-php-sdk/actions/runs/10664468962/job/29555788906?pr=103

And this is the output: https://github.com/fingerprintjs/fingerprint-pro-server-api-php-sdk/pull/103#issuecomment-2324256966

Output is "increase in 100%", because base branch has no clover output(didn't installed on base branch yet). This will be changed in the future PRs.